src/atoms/gv-input.js

```diff
@@ -298,11 +298,13 @@ export class GvInput extends InputElement(LitElement) {
 
       setTimeout(() => {
         const clipboardPopover = this.shadowRoot.querySelector('.clipboard__popover');
-        this.getInputElement().addEventListener('click', (e) => {
-          e.preventDefault();
-          e.stopPropagation();
-          clipboardPopover.click();
-        });
+        if (this.readonly) {
```

Comment:
don't know if it is related but I think there is a glitch on readonly story -> the value filled are not in the good format so the controls appears as invalid

![image](https://user-images.githubusercontent.com/47851994/106448010-79fbc000-6482-11eb-847a-590088c5e6c1.png)